panel/reactive.py

```diff
@@ -608,7 +617,7 @@ class SyncableData(Reactive):
     with the frontend via a ColumnDataSource.
     """
 
-    selection = param.List(default=[], doc="""
+    selection = param.List(default=[], class_=int, doc="""
```

Comment:
I believe the `class_` should be int. 

It will help when/ if we can automatically create *stubs* from `Parameterized` classes. Then the type stub can be inferred to `List[int]`.

Reply:
Yes, thanks!